inductiva/simulators/README.md

```diff
@@ -0,0 +1,235 @@
+range of features to solve anything from complex fluid flows involving chemical reactions, turbulence and heat transfer, to solid dynamics and electromagnetics.
+
+A single simulation via Inductiva API comprises several steps done via OpenFOAM - e.g., partitioning the domain, meshing, solvers and post-processing. Hence, to configure a simulation for OpenFOAM the user will need a set of configuration files that are spread across three folders:
+- `time`: containing individual files of data for particular fields, like initial values and boundary conditions that the user must specify. For example, for an initial condition at $t=0$ the initial conditions will be stored in the directory `0`.
+- `constant`: contains a full description of the case of the mesh, the physical properties for the application we are concerned, and the files describing some of the objects present inside the simulation and that need to be pre-processed to obtain a meshe.
+- `system`: contains all of the files that describe the simulation, including the solvers, the numerical parameters, and the output files. It must contain at least 3 files: `controlDict` where run control parameters are set including start/end time, time step and parameters for data output; `fvSchemes` where discretisation schemes used in the solution may be selected at run-time; and `fvSolution` where the equation solvers, tolerances and other algorithm controls are set for the run.
+
+All of these folders should be inside an input directory. Finally, to run a simulation the user needs to configure in list of dictionaries the commands that he wants to perform in the backend. Below, we run the [motorbike tutorial](https://github.com/OpenFOAM/OpenFOAM-8/tree/master/tutorials/incompressible/simpleFoam/motorBike) from OpenFOAM and show how this is done in practice.
+
+### Example 
+
+````python
+import inductiva
+
+inductiva.api_key = "YOUR_API_KEY"
+
+# Set simulation input directory
+input_dir = "motorbike_tutorial"
+
+# Set the simulation commands
+[{"cmd": "surfaceFeatures", "prompts": []}, {"cmd": "blockMesh", "prompts": []},
+{"cmd": "snappyHexMesh", "prompts": []}, {"cmd": "potentialFoam", "prompts": []},
+{"cmd": "simpleFoam", "prompts": []}]
+
+# Initialize the Simulator
+simulator = inductiva.simulators.OpenFOAM()
+
+# Run simulation with config files in the input directory
+task = simulator.run(input_dir=input_dir, commands=commands)
+````
+
+## SWASH simulator
+
+SWASH is a simulator that solves the shallow water equations and is used to simulate waves and currents in coastal waters and harbours, long waves in coastal regions and tidal inlets, and rapidly-varied flows around coastal structures. The simulator is configured by a single file with the extension `.sws` and extra files that contain information about the domain and bottom of the ocean - like a bathymetry file `.bot` -  are required to run the simulation.
```

Comment:
The simulator is configured using a single file with the `.sws` extension, and additional files containing information about the domain and the ocean floor, such as a bathymetry file with a `.bot` extension, are necessary for the simulation to run.